### PR TITLE
`[EXILED::API]` Adding MoveNetworkIdentityObject

### PR DESCRIPTION
**Pull request by NotZer0Two**
Lets the user move NetworkIdentities it can be bugged for the current culling system, in theory you can move object with a radius of 500m (Very small) and that will let them load in, could be useful for moving objects that you don't want in the facility to spawn or for example the Bowl of Candy

### CODE REVIEW

**Comment by NotZer0Two** [comment]
image

This is what you can do

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -4107,6 +4107,23 @@ public void SendFakeSceneLoading(string newSceneName)
         /// <param name="newSceneName">The new Scene the client will load.</param>
         public void SendFakeSceneLoading(ScenesType newSceneName) => SendFakeSceneLoading(newSceneName.ToString());
 
+        /// <summary>
+        /// Moves object for the player.
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Player.cs:4111]:
```suggestion
        /// Moves an object for the player.
```

File: Exiled.API/Features/Server.cs [hunk 2]
```diff
@@ -356,5 +356,25 @@ public static void ChangeSceneToAllClients(string newSceneName)
         /// </summary>
         /// <param name="scene">The new Scene the client will load.</param>
         public static void ChangeSceneToAllClients(ScenesType scene) => ChangeSceneToAllClients(scene.ToString());
+
+        /// <summary>
+        /// Moves object for all the players.
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Server.cs:361]:
```suggestion
        /// Moves an object for all players.
```

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Comment by NaoUnderscore** [comment]
Fix conflicts

**Review by Monaldcry7788 — APPROVED** [review]

**Review by xNexusACS — APPROVED** [review]

**Pull request merged**